applications/mne_scan/plugins/hpi/hpi.cpp

```diff
@@ -153,6 +153,7 @@ bool Hpi::start()
 
 bool Hpi::stop()
 {
+    requestInterruption();
```

Comment:
IF you ask for a request interruption here, you would need to include a check for interruption in the run() method.

Reply:
The loop in the run method of the hpi fit is `while(!isInterruptionRequested())`. Do you mean add more break points with that condition inside the loop?

Reply:
sry my bad. isInterruptionRequested() is used already indeed.